### PR TITLE
test: integration test

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  public void getPolicyTest() {
+    Policy policy = RESOURCE_MANAGER.getPolicy(PROJECT);
+    assertNotNull(policy);
+    assertNotNull(policy.getEtag());
+    assertEquals(1, policy.getVersion());
+
+    /* policy bindings */
+    Map<Role, Set<Identity>> policyBindings = policy.getBindings();
+    assertFalse(policy.getBindings().isEmpty());
+    assertTrue(policyBindings.size() > 0);
```
Comment (fazunenko) [inline src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:60]: one check is redundant
it's enough to check either that policyBindings.size() > 0 or policyBindings.isEmpty() == false

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    Policy policy = resourceManager.getPolicy(PROJECT);
+    assertNotNull(policy);
+    assertNotNull(policy.getEtag());
+    assertEquals(1, policy.getVersion());
+
+    /* policy bindings */
+    Map<Role, Set<Identity>> policyBindings = policy.getBindings();
+    assertFalse(policy.getBindings().isEmpty());
+    assertTrue(policyBindings.size() > 0);
```
Comment (fazunenko) [inline src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:79]: the same here

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  @Test
+  public void testPermissionsTest() {
+    String getPermission = "resourcemanager.projects.get";
+    String deletePermission = "resourcemanager.projects.delete";
+    List<Boolean> booleanList =
+        RESOURCE_MANAGER.testPermissions(
+            PROJECT, ImmutableList.of(getPermission, deletePermission));
+    assertFalse(booleanList.isEmpty());
+    assertTrue(booleanList.size() > 0);
+    assertEquals(ImmutableList.of(true, false), booleanList);
```
Comment (fazunenko) [inline src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:90]: the last check is enough here

**Review by pmakani — APPROVED** [review]
LGTM

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java (hunk 1 above)
Comment (athakor) [reply src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:60]: done

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java (hunk 2 above)
Comment (athakor) [reply src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:79]: done

File: src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java (hunk 3 above)
Comment (athakor) [reply src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:90]: done

Comment (dmitry-fa) [reply src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:90]: it's enough to check only:
assertEquals(ImmutableList.of(true, false), booleanList);

Comment (athakor) [reply src/test/java/com/google/cloud/resourcemanager/it/ITSystemTest.java:90]: yes

**Pull request closed without merging**